templates/register.html

```diff
@@ -22,7 +22,7 @@ <h2 class="auth-main__title">
 
 <p class="auth-main__content--small">
   Already have an account?
-  <a href="#">Login</a>
+  <a href="login">Login</a>
```

Comment:
Missing the forward slash, no?

```suggestion
  <a href="/login">Login</a>
```

Reply:
Good catch. Fixed in 096a130cd981d6647567c8dce1f60190adc893df